src/entities/task.entity.ts

```diff
@@ -49,4 +50,17 @@ export class Task {
   public area: TaskArea;
   public assignee: string;
   public isBodyVisible: boolean;
+  public tags: string[];
+}
+
+export const getTaskTags = (task: Task): string[] => {
```

Comment:
It looks like an Ember computed property hahah xD

Reply:
Indeed It looks like 😅

Don't know If this is good or bad though.

And thinking about this now I wonder If It wouldn't have been better to use javascript's `getter` feature, like:

```js
...

get tags(task: Task): string[] {
    switch (task.type) {
        ...
    }
}
```